cropharvest/eo/eo.py

```diff
@@ -231,34 +248,50 @@ def _bounding_box_from_centre(
         max_lat, min_lat = mid_lat + deg_lat, mid_lat - deg_lat
         max_lon, min_lon = mid_lon + deg_lon, mid_lon - deg_lon
 
-        return ee.Geometry.Polygon(
+        return (min_lon, min_lat, max_lon, max_lat), ee.Geometry.Polygon(
             [[[min_lon, min_lat], [min_lon, max_lat], [max_lon, max_lat], [max_lon, min_lat]]]
         )
 
+    @classmethod
     def _labels_to_polygons_and_years(
-        self, labels: geopandas.GeoDataFrame, surrounding_metres: int
-    ) -> List[Tuple[ee.Geometry.Polygon, str, int]]:
+        cls, labels: geopandas.GeoDataFrame, surrounding_metres: int
+    ) -> List[Tuple[ee.Geometry.Polygon, str, date, date]]:
 
         output: List[ee.Geometry.Polygon] = []
 
         print(f"Exporting {len(labels)} labels")
 
         for _, row in tqdm(labels.iterrows()):
+            latlons, bounding_box = cls._bounding_box_from_centre(
+                mid_lat=row["lat"], mid_lon=row["lon"], surrounding_metres=surrounding_metres
+            )
+
+            try:
```

Comment:
Why try-except instead of if else?

Reply:
:shrug: I can change it!

Reply:
Never mind, I was wrong try except is the way to go: https://stackoverflow.com/questions/11360858/what-is-the-eafp-principle-in-python